test/helpers/common.ts

```diff
@@ -18,9 +30,96 @@ export async function checkTimeSliceForUpgrades(
   return { result: !onChainRt.eq(currentVersion), specVersion: onChainRt };
 }
 
-// Sort dict by key
-export function sortObjectByKeys(o) {
-  return Object.keys(o)
+/**
+ * Sorts the keys of an object and returns a new object with the same values,
+ * but with keys in lexicographical order.
+ *
+ * @export
+ * @param {Record<string, any>} unsortedObject - The object with unsorted keys.
+ * @returns {Record<string, any>} - The new object with keys sorted.
+ */
+export function sortObjectByKeys(unsortedObject: Record<string, any>): Record<string, any> {
```

Comment:
Maybe we should use a standard library for small useful functions instead of writing our own.